sonar-java-plugin/src/main/resources/org/sonar/l10n/java/rules/java/S6926.json

```diff
@@ -0,0 +1,24 @@
+{
+  "title": "Bluetooth should be configured to use low power",
+  "type": "CODE_SMELL",
+  "status": "ready",
+  "remediation": {
+    "func": "Constant\/Issue",
+    "constantCost": "5min"
+  },
+  "tags": [
+    "android",
+    "sustainability"
+  ],
+  "defaultSeverity": "Major",
+  "ruleSpecification": "RSPEC-6926",
+  "sqKey": "S6926",
+  "scope": "Main",
+  "quickfix": "unknown",
+  "code": {
+    "impacts": {
+      "MAINTAINABILITY": "LOW"
```

Comment:
Is it really maintainability that is impacted? Shouldn't it be sustainability?

Reply:
Since there is no sustainability impact defined, it was agreed with Alex to set maintainability=LOW and have a sustainability tag.